release/_releases_json.sh

```diff
@@ -31,6 +41,26 @@ function _merge_json_snippets {
 	fi
 }
 
+function _process_new_product {
+	local releases_json="${_RELEASE_ROOT_DIR}/release-data/promotion/files/0000-00-00-releases.json"
+
+	if [ ! -f "${releases_json}" ]
+	then
+		lc_log INFO "Downloading https://releases.liferay.com/releases.json to ${releases_json}."
+
+		lc_download https://releases.liferay.com/releases.json "${releases_json}"
+	fi
+
+	if (grep "${LIFERAY_RELEASE_VERSION}" "${releases_json}")
+	then
+		lc_log INFO "The version ${LIFERAY_RELEASE_VERSION} is already in releases.json"
+
+		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
+	else
+		_process_product_version "${LIFERAY_RELEASE_PRODUCT_NAME}" "${LIFERAY_RELEASE_VERSION}"
```

Comment:
I realised that we don't need the `local product_name="${1}"` var in this function because here we only check the version from the input so we can just use `"${LIFERAY_RELEASE_PRODUCT_NAME}"` instead